zulint/custom_rules.py

```diff
@@ -160,13 +165,8 @@ def check_file_for_long_lines(self,
             else:
                 line_length = len(line)
             if (line_length > max_length and
-                '# type' not in line and 'test' not in fn and 'example' not in fn and
-                # Don't throw errors for markdown format URLs
-                not re.search(r"^\[[ A-Za-z0-9_:,&()-]*\]: http.*", line) and
-                # Don't throw errors for URLs in code comments
-                not re.search(r"[#].*http.*", line) and
-                not re.search(r"`\{\{ api_url \}\}[^`]+`", line) and
-                    "# ignorelongline" not in line and 'migrations' not in fn):
+                not re.findall(r'|'.join(self.exclude_max_length_fns), fn) and
+                not re.findall(r'|'.join(self.exclude_max_length_line_patterns), line)):
```

Comment:
This broke the check because `re.findall("", …)` is truthy when either exclude list is empty.

I’m removing it completely in #28.